test/index.js

```diff
@@ -0,0 +1,14 @@
+import assert     from 'assert'
+import superagent from 'superagent'
+
+require('../lib/index')
+
+describe('/version', () => {
+  it('returns the current version', (done) => {
+    superagent.get('http://localhost:8000/version')
```

Comment:
Requiring `lib/index` on line 4 should automatically start the server. I'm assuming that as the app becomes more complex, this might cause some confusion and require that I start the server with a clearer command.